tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
+    "noUnusedLocals": false,
```

Comment:
```suggestion
```

Reply:
I think we should leave as it's counter productive to have it on. Lint guarantees that effect